src/components/BalanceInfo/BalanceInfo.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import ReferenceStore from '../../stores/referenceStore';
+import Select from '../Select/select';
+import './balance-info.css';
+import {BalanceInfoProps} from './index';
+
+const StyledSelect = styled(Select)`
+  .Select-control {
+    background-color: #333333 !important;
+
+    .Select-value {
+      .Select-value-label {
+        color: #fff !important;
+      }
+    }
+
+    .Select-input {
```

Comment:
just as a side note – this is not the _styled-component way_ :) for now it's fine but plz check how it's intended to use at https://www.styled-components.com/